freud/environment.pyx

```diff
@@ -409,9 +405,104 @@ cdef class LocalDescriptors(PairCompute):
                     negative_m=self.negative_m, mode=self.mode)
 
 
-cdef class MatchEnv(Compute):
-    R"""Clusters particles according to whether their local environments match
-    or not, according to various shape matching metrics.
+def minimizeRMSD(box, ref_points, points, registration=False):
+    R"""Get the somewhat-optimal RMSD between the set of vectors ref_points
```

Comment:
Don't say "somewhat-optimal" -- it's at least a local minimum, right?

Reply:
Yes, it finds a local minimum in some region of the landscape, but there's some random number generation in there, so running it multiple times can give you different results. So it seems pretty inaccurate to remove the extra qualifications.